### PR TITLE
Resolve updated codecheck issues

### PR DESCRIPTION
**Pull request by adlarkin**
This PR is meant to resolve new codecheck issues introduced by https://github.com/ignitionrobotics/ign-cmake/pull/117.

From my understanding, the plan is to forward port the fixes here to all other ignition versions once https://github.com/ignitionrobotics/ign-cmake/pull/117 is merged.

Signed-off-by: Ashton Larkin <ashton@openrobotics.org>

### CODE REVIEW

File: src/plugins/image_display/ImageDisplay.cc [hunk 1]
```diff
@@ -15,16 +15,21 @@
...
+
+#include <QQuickImageProvider>
 
 #include <ignition/common/Console.hh>
 #include <ignition/common/Image.hh>
 #include <ignition/plugin/Register.hh>
 #include <ignition/transport/Node.hh>
 
 #include "ignition/gui/Application.hh"
-#include "ImageDisplay.hh"
```
Comment (chapulina) [inline src/plugins/image_display/ImageDisplay.cc:27]: This change makes the local code checker fail:

```
./src/plugins/image_display/ImageDisplay.cc:25:  Found C system header after C++ system header. Should be: ImageDisplay.h, c system, c++ system, other.  [build/include_order] [4]
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/plugins/image_display/ImageDisplay.cc (hunk 1 above)
Comment (adlarkin) [reply src/plugins/image_display/ImageDisplay.cc:27]: I believe the error is actually coming from `#include <QQuickImageProvider>` (it looks like the local checker interprets this as a C system header). I'm working on a fix.

Comment (adlarkin) [reply src/plugins/image_display/ImageDisplay.cc:27]: Should be fixed now in https://github.com/ignitionrobotics/ign-gui/pull/144/commits/3b1494cf12ae07bc1f53429fca8cc1ccd48e4ce1.

**Review by chapulina — APPROVED** [review]
Looks good now :+1:

**Pull request merged**